fedimint-core/src/api.rs

```diff
@@ -726,13 +726,13 @@ impl JsonRpcClient for WsClient {
         return WsClientBuilder::default()
             .use_webpki_rustls()
             .max_concurrent_requests(u16::MAX as usize)
-            .build(url_to_string_with_default_port(url)) // Hack for default ports, see fn docs
+            .build(url.as_str()) // Hack for default ports, see fn docs
             .await;
 
         #[cfg(target_family = "wasm")]
         WsClientBuilder::default()
             .max_concurrent_requests(u16::MAX as usize)
-            .build(url_to_string_with_default_port(url)) // Hack for default ports, see fn docs
+            .build(url.as_str()) // Hack for default ports, see fn docs
```

Comment:
Stale comments now?

Reply:
Yes, should be removed

Reply:
done [ebed3b7](https://github.com/fedimint/fedimint/pull/3454/commits/ebed3b7e8ccb5d5c319fdf2455a5b5c0b41eec49)